resources/fetching/fetchers/azure/assets_fetcher.go

```diff
@@ -105,6 +108,11 @@ func (f *AzureAssetsFetcher) Fetch(ctx context.Context, cycleMetadata cycle.Meta
 	if err != nil {
 		f.log.Errorf("Error fetching subscription information: %v", err)
 	}
+
+	if err := f.enrichStorageAccountAssets(ctx, cycleMetadata, subscriptions, assets); err != nil {
```

Comment:
I think that we should wrap this with a general enrichment wrapper in order to implement future enrichments inside it, but this refactoring can be done with the implementation of the next enricher